### PR TITLE
Add nuspec template file

### LINKED ISSUE #180: Add nuspec file
To the folder `.ci/BHoMBot/Nuget` - in the form of `{repository_name}.nuspec` to help the bot automatically produce nugets of this repository.

### PR DESCRIPTION
**Pull request by FraserGreenroyd**
Fixes #180

### CODE REVIEW

**Review by kprusicka — DISMISSED** [review]
The nuspec template file is indeed added.

File: .ci/BHoMBot/Nuget/BHoM.Interop.Mongo.nuspec [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    <icon>icon.png</icon>
+    <readme>readme.md</readme>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <description>Interoperability with Mongo for BHoM - more information available at https://github.com/BHoM/Mongo_Toolkit</description>
+    <releaseNotes></releaseNotes>
+    <copyright></copyright>
+    <tags>BHoM interop mongo aec</tags>
+    <title></title>
+    <dependencies>
+      <group targetFramework=".NetStandard2.0">
```
Comment (FraserGreenroyd) [inline .ci/BHoMBot/Nuget/BHoM.Interop.Mongo.nuspec:18]:
```suggestion
      <group targetFramework="netstandard2.0">
```

**Review by kprusicka — APPROVED** [review]
Changes reviewed, looks good.

**Review by kprusicka — APPROVED** [review]
Looks amazing to me

**Review by Dhorix — APPROVED** [review]
Reviewed, Looks great.

**Comment by FraserGreenroyd** [comment]
@BHoMBot this is a DevOps instruction. I am requesting neutral checks on: code-compliance, copyright-compliance, core, documentation-compliance, installer, project-compliance, ready-to-merge, serialisation, versioning, dataset-compliance, unit-tests

**Comment by FraserGreenroyd** [comment]
@BHoMBot this is a DevOps instruction. I am requesting neutral checks on: null-handling

**Pull request merged**